src/utils/MzMLSplitter.cpp

```diff
@@ -86,7 +86,7 @@ class TOPPMzMLSplitter :
     setMinInt_("size", 0);
     registerStringOption_("unit", "<choice>", "MB", "Unit for 'size' (base 1024)", false);
     setValidStrings_("unit", ListUtils::create<String>("KB,MB,GB"));
-    registerFlag_("precursor", "Make sure precursor spectra end up in the same part as their fragment spectra");
+    registerFlag_("precursor", "The precursor spectrum and its corresponding MSn spectra will be saved in the same file");
```

Comment:
I would rename this parameter. Also, it is not clear how setting the flag relates to the other options.